### ISSUE TITLE
Error when trying to perform tasks and checks if return code is successful

### ISSUE THREAD

**Issue by mvoto** [r7com/buildable#2]
Error Trace:

```
NoMethodError: undefined method `success?' for nil:NilClass
rake aborted!
NoMethodError: undefined method `success?' for nil:NilClass
/var/lib/jenkins/jobs/gatekeeper/workspace/vendor/ruby/2.1.0/gems/buildable-1.2.4/lib/buildable/recipes/build.rb:23:in `block (3 levels) in <module:Recipe>'
```

Using rbenv's ruby 2.1.2p95 version
I did a workaround on a forked repo(and a debug mode commit): https://github.com/mvoto/buildable/commit/e961588ceac7529082ef2e552d8878da0323591d as suggested on [this article](http://www.shanison.com/2010/09/11/ptychildexited-exception-and-ptys-exit-status/)

Not sure if this is the best approach, any concerns about it ?

**Comment by alexandreprates** [comment]
fixed in version 1.2.7

**Comment by mvoto** [comment]
Indeed, thanks !
